<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>11</y>
     <width>274</width>
     <height>269</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="currentTrack">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <italic>true</italic>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="btn_add">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="buttons.qrc">
          <normaloff>:/buttons/btn_add.png</normaloff>:/buttons/btn_add.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btn_previus">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="buttons.qrc">
          <normaloff>:/buttons/btn_previous.png</normaloff>:/buttons/btn_previous.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btn_play">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="buttons.qrc">
          <normaloff>:/buttons/btn_play.png</normaloff>:/buttons/btn_play.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btn_pause">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="buttons.qrc">
          <normaloff>:/buttons/btn_pause.png</normaloff>:/buttons/btn_pause.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btn_next">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="buttons.qrc">
          <normaloff>:/buttons/btn_next.png</normaloff>:/buttons/btn_next.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btn_stop">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="buttons.qrc">
          <normaloff>:/buttons/btn_stop.png</normaloff>:/buttons/btn_stop.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="playlistView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="buttons.qrc"/>
 </resources>
 <connections/>
</ui>
